views/comp_image.ejs

```diff
@@ -0,0 +1,29 @@
+<%- include("partials/head") %>
+    <h1 id="image">Image</h1>
+    <h2 id="responsive-image">Responsive image</h2>
+    <div class="component__container">
+        <div style="width: 300px; border-radius: 0.25rem; overflow: hidden;">
+            <img class="responsive-img" src="https://picsum.photos/300" alt="">
```

Comment:
You can remove inline styling

Reply:
these styles are specific to this page and component, since i have different pages for each component having them on all pages does not make sense. What do you suggest ?